src/experiments/logins/schema.json

```diff
@@ -199,6 +199,32 @@
       }
     ],
     "functions": [
+      {
+        "name": "getLoginSavingEnabled",
+        "type": "function",
+        "description": "For the given origin, gets a boolean flag which determines whether login saving is enabled",
+        "async": true,
+        "parameters": [{
+          "name": "aHost",
+          "type": "string",
+          "description": "The origin for which to check the status of login saving"
+        }]
+      },
+      {
+        "name": "setLoginSavingEnabled",
+        "type": "function",
+        "description": "For the given origin, uses the given value to set a boolean flag which determines whether login saving is enabled",
```

Comment:
The IDL docs may be clearer here. Also, be sure to mention cases in which the returned promise is rejected, which I think is just if the URL is malformed.